code/modules/mob/living/carbon/human/combat.dm

```diff
@@ -27,51 +27,42 @@
 			return G.afterattack(target, src, "struggle" = 1)
 
 /mob/living/carbon/human/disarm_mob(mob/living/target)
-	src.attack_log += text("\[[time_stamp()]\] <font color='red'>Disarmed [target.name] ([target.ckey])</font>")
-	target.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been disarmed by [src.name] ([src.ckey])</font>")
+	add_logs(src, target, "disarmed", admin = (src.ckey && target.ckey) ? TRUE : FALSE) //Only add this to the server logs if both mobs were controlled by player
 
-	log_attack("[src.name] ([src.ckey]) disarmed [target.name] ([target.ckey])")
+	if(target.disarmed_by(src))
+		return
 
 	var/datum/organ/external/affecting = get_organ(ran_zone(zone_sel.selecting))
-	if(target.disarmed_by(src))
+	if(prob(40))
+		playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
+		visible_message("<span class='danger'>[src] has attempted to disarm [target]!</span>")
 		return
 
-	var/randn = rand(1, 100)
-	if(randn <= 25)
+	if(prob(40)) //40% to push
```

Comment:
I'll have to clarify that in the comment

There's a 60% chance of not missing and a 40% chance of stunning (if you don't miss). 0.6 * 0.4 = 24% chance of stunning per click, roughly same as before